src/SnackbarItem/SnackbarItem.tsx

```diff
@@ -110,7 +110,7 @@ const SnackbarItem: React.FC<SnackbarItemProps> = ({ classes: propClasses, ...pr
 
     const handleEntered: TransitionHandlerProps['onEntered'] = () => {
         if (props.snack.requestClose) {
-            handleClose(null, REASONS.INSTRCUTED);
+            handleClose(null, REASONS.INSTRUCTED);
```

Comment:
This is the only line change needed to resolve #454 . I'll happily merge the PR once other changes are reverted.